dbms/src/Debug/dbgFuncMisc.cpp

```diff
@@ -97,4 +98,13 @@ void dbgFuncSearchLogForKey(Context & context, const ASTs & args, DBGInvoker::Pr
     else
         output("Invalid");
 }
+
+void dbgFuncTriggerGlobalPageStorageGC(Context & context, const ASTs & /*args*/, DBGInvoker::Printer /*output*/)
+{
+    auto global_storage_pool = context.getGlobalStoragePool();
+    if (global_storage_pool)
+    {
+        global_storage_pool->meta()->gc();
```

Comment:
what about also calling GC on the global data and log?

Reply:
added.